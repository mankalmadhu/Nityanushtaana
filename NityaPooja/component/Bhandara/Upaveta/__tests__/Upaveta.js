import 'react-native';
import React from 'react';
import ಉಪವೀತ from '../Upaveta';

import Renderer from 'react-test-renderer';

jest.mock('../../../Avarana/Avarana',()=>'ಆವರಣ');
jest.mock('../../../Vishaya/Sheershike',()=>'ಶೀರ್ಷಿಕೆ');
jest.mock('../../../Vishaya/Purvanga',()=>'ಪೂರ್ವಾ೦ಗ');
jest.mock('../../../Vishaya/Viniyoga',()=>'ವಿನಿಯೋಗ');
jest.mock('../../../Vishaya/Vidhana',()=>'ವಿಧಾನ');

test("ಉಪವೀತ" , () => {

expect(Renderer.create(
    <ಉಪವೀತ />
).toJSON()).toMatchSnapshot();

});