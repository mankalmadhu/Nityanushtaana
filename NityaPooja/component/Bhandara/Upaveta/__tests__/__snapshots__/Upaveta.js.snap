// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ಉಪವೀತ 1`] = `
<ಆವರಣ>
  <ಶೀರ್ಷಿಕೆ>
    ಉಪವೀತ
  </ಶೀರ್ಷಿಕೆ>
  <ವಿನಿಯೋಗ>
    ತಂ ಯ॒ಜ್ಞಂ ಬ॒ರ್ಹಿಷಿ॒ ಪ್ರೌಕ್ಷನ್॑ । ಪುರು॑ಷಂಜಾ॒ತಮ॑ಗ್ರ॒ತಃ । ತೇನ॑ ದೇ॒ವಾ ಅಯ॑ಜಂತ । ಸಾ॒ಧ್ಯಾ ಋಷ॑ಯಶ್ಚ॒ಯೇ ।
  </ವಿನಿಯೋಗ>
  <ಪೂರ್ವಾ೦ಗ>
    ಶ್ರೀ ಇಷ್ಟ ದೇವತಾಭ್ಯೋ ನಮಃ । ಉಪವೀತ೦ ಸಮರ್ಪಯಾಮಿ ॥
  </ಪೂರ್ವಾ೦ಗ>
</ಆವರಣ>
`;
